infra/main.bicep

```diff
@@ -114,6 +115,7 @@ module speechRecognizer 'core/ai/cognitiveservices.bicep' = {
     kind: 'SpeechServices'
     location: location
     tags: tags
+    customSubDomainName: speechSubdomain
```

Comment:
@Welasco I've added this bit to the infra and would love your feedback. To support Entra auth with Speech Services, a custom subdomain name was required.

Reply:
@jmatthiesen, looks good for me. I reviewed the change and where it's affecting and looks good.